packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1332,16 +1327,17 @@ curl -v -X GET \
 ]
 ```
 
-
-
 #### List Users with a filter
 
 Lists all users that match the filter criteria. To ensure optimal performance, Okta recommends using a [search parameter](#list-users-with-search) instead of a filter.
 
 This operation:
 
-* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `filter=lastUpdated gt "2013-06-01T00:00:00.000Z"` is encoded as `filter=lastUpdated%20gt%20%222013-06-01T00:00:00.000Z%22`.
-* Supports only a limited number of properties: `status`, `lastUpdated`, `id`, `profile.login`, `profile.email`, `profile.firstName`, and `profile.lastName`.
+- Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `filter=lastUpdated gt "2013-06-01T00:00:00.000Z"` is encoded as `filter=lastUpdated%20gt%20%222013-06-01T00:00:00.000Z%22`.
+- Supports the following limited number of properties: `status`, `lastUpdated`, `id`, `profile.login`, `profile.email`, `profile.firstName`, and `profile.lastName`.
+- Supports only the equal `eq` operator from the standard Okta API filtering semantics, except in the case of the `lastUpdated` property which can use the inequality operators (`gt`, `ge`, `lt`, and `le`).
```

Comment:
```suggestion
- Supports only the equal `eq` operator from the standard Okta API filtering semantics, except in the case of the `lastUpdated` property that can use the inequality operators (`gt`, `ge`, `lt`, and `le`).
```

Reply:
Hm, I think `which` is more correct here.  One of my thesis reviewers back in the day taught me a handy rule for this one:  Use `which` if you _could_ put a comma in front of it.  Here we are talking about the `lastUpdated` property, which can blah blah blah.  The comma can be optional if it reads better without it.  But if you use "the `lastUpdated` property that blah blah blah" it means you are differentiating this property from some _other_ `lastUpdated` property that doesn't blah blah blah.

Reply:
I'm going with `which` here as well, but it was missing the comma.